.jhipster/modules/jhi-hooks.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Where does this file come from ? as I don't see an existing module for Cypress?

Reply:
This file come from a custom module from https://github.com/clementdessoude/generator-jhipster-cypress but we change a lot of things here. So, we can delete this file as we will not use this custom module.